tests/default.nf.test

```diff
@@ -12,15 +12,16 @@ nextflow_pipeline {
         }
 
         then {
-            def file_complement = UTILS.getAllFilesFromDir(params.outdir, null, UTILS.exclusionRegexesForUnstableFileNames)
-            def snapshottable_files = UTILS.getAllFilesFromDir(params.outdir, UTILS.snapshottablePatterns, UTILS.exclusionRegexesForUnstableFileContents)
+            def path_not_snapshottable = UTILS.getAllFilesFromDir(params.outdir, null, UTILS.exclusionRegexesForUnstableFileNames)
```

Comment:
The complete file complement includes things which are snapshottable, hence the `file_complement` naming. I think `path_not_snapshottable` is confusing.

Reply:
fully agreeing with you, I'll try a better variable name

Reply:
what do you think about path_from_outdir

Reply:
What was wrong with file_complement?

Reply:
Just not super sure about the complement definition.
But I'm not too fussed about it, so happy to switch it back to what the variables names you came up with.
I just needed something I'm fully onboard with while debugging, now that I'm done with that part and just working on the regex, I'm ok with whatever.